app/components/memorable_date_component.html.erb

```diff
@@ -30,17 +30,18 @@
                     maxLength: 2,
                     aria: {
                       invalid: false,
-                      describedby: [
-                        "validated-field-error-#{unique_id}",
-                        "validated-field-hint-#{unique_id}",
-                      ],
+                      describedby: "validated-field-error-#{unique_id}",
+                      labelledby: 'month_hint_id',
                     },
                     value: month,
                   },
                   error: { id: "validated-field-error-#{unique_id}" },
                   required: required,
                 ) %>
           <% end %>
+          <span id="month_hint_id" class="usa-sr-only" aria-hidden="true">
```

Comment:
I think I understand why it's needed (so that the user won't be able to navigate to it through screen reader content navigation), but are there any issues where a screen reader may refuse to read this as a label because it's removed from the accessibility tree?

Would it be better to use `display: none;` instead? I've seen some documentation which indicates that this should work as expected:

>The `aria-labelledby` property is useful in a wide variety of situations because: [...] It incorporates content from elements regardless of their visibility, i.e., it even includes content from elements with the HTML `hidden` attribute, CSS `display: none`, or CSS `visibility: hidden` in the calculated name string.

https://w3c.github.io/aria-practices/#naming_with_aria-labelledby

```suggestion
          <span id="month_hint_id" class="display-none">
```

Reply:
testing it out with voiceover it has the same behavior so i'll update all of the inputs to use display-none instead as suggested here.